diff --git a/vendor/raylib/raylib.odin b/vendor/raylib/raylib.odin
index 34a803b..5be605d 100644
--- a/vendor/raylib/raylib.odin
+++ b/vendor/raylib/raylib.odin
@@ -100,8 +100,11 @@ MAX_TEXT_BUFFER_LENGTH :: #config(RAYLIB_MAX_TEXT_BUFFER_LENGTH, 1024)
 #assert(size_of(rune) == size_of(c.int))
 
 RAYLIB_SHARED :: #config(RAYLIB_SHARED, false)
+RAYLIB_SYSTEM :: #config(RAYLIB_SYSTEM, false)
 
-when ODIN_OS == .Windows {
+when RAYLIB_SYSTEM {
+	foreign import lib "system:raylib"
+} else when ODIN_OS == .Windows {
 	@(extra_linker_flags="/NODEFAULTLIB:" + ("msvcrt" when RAYLIB_SHARED else "libcmt"))
 	foreign import lib {
 		"windows/raylibdll.lib" when RAYLIB_SHARED else "windows/raylib.lib" ,
